_chapters/chapter_01.md

```diff
@@ -166,24 +182,31 @@ layout: rails_tutorial
 {% aside %}
   Not sure what a database is? Don't worry, we'll fill you in :)
 
-  Brief description with external resources if we think this is necessary.
+  There are many different types of databases but the one we'll be working with here - and that's the most similar to what you've probably worked with in the past is called a Relational Database. There are many types of *Relational Database Management Systems* (RDMS's for short) and SQLite is just one of them. 
```

Comment:
This is good, but would it be better to have a brief description of databases? I'm not sure what that description would be, but maybe something like "A database is made up of tables where you store stuff...kinda like an Excel spreadsheet"?

Reading the description we have now doesn't really answer the question of "What is a database".